README.md

```diff
@@ -99,7 +99,7 @@ official compatibility chart below.
 
 ## Usage
 
-`kubernetes-deploy <app's namespace> <kube context>`
+`krane <app's namespace> <kube context>`
```

Comment:
`krane deploy <app's namespace> <kube context>` looks like this needs to be fixed up in a few places in this file, it's a bit easier to see it in the 'rich diff' mode.